### DISCUSSION TITLE
Block promotional emails

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1606] [Feature Requests]
Please implement feature to have option to block promotional emails for aliases. Firefox relay has this feature and it is very helpful to stop inbox clutter.

**Comment by HadetTheUndying** [comment]
I would love to see this implemented as well. Currently i have to setup strict filtering rules to get the emails to at least be put into a folder.

**Comment by gdonval** [comment]
Just to make things clear, what is the problem with the existing feature of blocking a contact sending promotional emails? I.e. usually important stuff will come from an email address like `order@...` and spam will come from `news@...` so you can just block `news@...` and be done with it. 

Or are you asking about Simple Login reading the content of the email and putting it into spam if they trigger spam filters? In that case, without trying to be dismissive, what's wrong with using your final email service's spam filter?
